ArduCopter/Log.cpp

```diff
@@ -114,6 +114,8 @@ struct PACKED log_Nav_Tuning {
     float    vel_y;
     float    desired_accel_x;
     float    desired_accel_y;
```

Comment:
We should change this to target as well. Also, I was a bit confused by the commit message *Copter: add accel target to NTUN log msg*; if the desired is the target accel, then it isn't adding it as it was already there 🙂